Core/HLE/sceKernelThread.h

```diff
@@ -77,6 +77,8 @@ struct SceKernelSysClock {
 	u32_le hi;
 };
 
+static const int ERROR_KERNEL_WAIT_CAN_NOT_WAIT = 0x800201a7;
```

Comment:
We have `SCE_KERNEL_ERROR_CAN_NOT_WAIT` already - we shouldn't define a new value here.

`SCE_KERNEL_ERROR_ILLEGAL_ARGUMENT` for the other.

-[Unknown]